src/marqo/tensor_search/validation.py

```diff
@@ -14,6 +14,7 @@
 from enum import Enum
 import jsonschema
 from marqo.tensor_search.models.settings_object import settings_schema
+from marqo.tensor_search.models.mappings_object import mappings_schema, multimodal_combination_schema
 
```

Comment:
Make sure there is only one `mappings_validation` and only one `multimodal_combination_mappings_validation`

Reply:
done